packages/ui-decorator/src/decorator.ts

```diff
@@ -22,19 +22,24 @@
  * SOFTWARE.
  */
 
+import type { ComponentClass } from 'react'
+
 /**
  * ---
  * category: utilities
  * ---
  * Wraps (decorates) a React component to add additional functionality
  *
  * @module decorator
- * @param {Function} decorator a function that returns a decorated component
- * @returns {Class} a decorated component class
```

Comment:
These types are gone from the doc page too. (And at the other methods where you deleted them too :D)